app/package.json

```diff
@@ -33,6 +33,7 @@
     "@chakra-ui/next-js": "^2.2.0",
     "@chakra-ui/react": "^2.8.2",
     "@huggingface/inference": "^2.8.0",
+    "@next/env": "^14.2.5",
```

Comment:
Let's move this to a separate PR, as it causes issues in our CI pipeline it seems

Reply:
Or commit the required update for `package-lock.json`, then we can keep it :sweat_smile: